cns/service/main.go

```diff
@@ -418,7 +426,7 @@ func main() {
 		logger.Errorf("[Azure CNS] Cannot disable telemetry via cmdline. Update cns_config.json to disable telemetry.")
 	}
 
-	cnsconfig, err := configuration.ReadConfig()
+	cnsconfig, err := configuration.ReadConfig(cmdLineConfigPath)
```

Comment:
logging cmdLineConfigPath will be helpful for debugging

Reply:
Got it